core/shared/src/main/scala/scalaz/zio/Semaphore.scala

```diff
@@ -10,6 +10,8 @@ import scala.collection.immutable.{ Queue => IQueue }
 
 final class Semaphore private (private val state: Ref[State]) extends Serializable {
 
+  type AcquireTasks = (IO[Nothing, Unit], IO[Nothing, Unit])
```

Comment:
Let's move that into the companion.

Reply:
@regiskuckaertz It seemed right to move AcquireTaks to internals, did I do it right?

Reply:
Yep